libs/@blockprotocol/type-system/crate/README.md

```diff
@@ -13,10 +13,10 @@
 
```

Comment:
Can't quite comment on it, but do we require `cargo-make` at all any more?

Reply:
I removed it from the action, thanks for flagging